calicoctl/commands/version.go

```diff
@@ -130,3 +130,38 @@ Description:
 
 	return err
 }
+
+func VersionMismatch(args []string) error {
+
+	config := constants.DefaultConfigPath
+
+	// We need to "look ahead" to see if --config has been passed in the args
+	for i := range args {
+		if args[i] == "--config" && i < len(args)-1 {
+			config = args[i+1]
+		}
+	}
+
+	client, err := clientmgr.NewClient(config)
+	if err != nil {
+		return fmt.Errorf("Unable to verify version mismatch: %w", err)
```

Comment:
you may want to give these error messages something a little different so you can tell what happened apart in case you're debugging based on the error message

Reply:
Will do!